src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public Ball getRandomBall() {
+        String color = String.valueOf(getRandomColor());
+        Random random = new Random();
+        int index = random.nextInt(100 + 1);
+        Ball ball = new Ball(color, index);
+        return ball;

```

Comment:
Don’t create temporary variables if you can directly return value from the method
[Common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#dont-create-temporary-variables-if-you-can-directly-return-value-from-the-method)